src/components/managers/products/listProducts/filterProdcuts/initialFilterStates.ts

```diff
@@ -0,0 +1,19 @@
+import { GetProductsPagedRequest } from "api/proto-http/admin";
+
+export const initialFilter: GetProductsPagedRequest = {
```

Comment:
why do we call this const as initialFilter , if it's some kind of mocked request ? wrong naming here